### ISSUE TITLE
No detections with Kitti dataset

### ISSUE THREAD

**Issue by lovehell** [eric612/MobileNet-YOLO#150]
Hi,

I've converted the kitti (9 classes) dataset into LMDB files and trained a YOLOV3. The loss is surprisingly low after hundreds of iterations (loss = 10^-5) and when I run the network there are no detection, even with a confidence threshold of 0. 

I've changed the number of classes in the network (as raised in other issue tickets) and double checked my files. 

Do you have any clues on what's going on ? Is it related to bad parameters or corrupted LMDB files ? I've also generated new anchor boxes by the way.

Thanks

**Comment by hitzht** [comment]
> Hi,
> I've converted the kitti (9 classes) dataset into LMDB files and trained a YOLOV3. The loss is surprisingly low after hundreds of iterations (loss = 10^-5) and when I run the network there are no detection, even with a confidence threshold of 0.
> I've changed the number of classes in the network (as raised in other issue tickets) and double checked my files.
> Do you have any clues on what's going on ? Is it related to bad parameters or corrupted LMDB files ? I've also generated new anchor boxes by the way.
> I've also noticed that detection_eval is equal to 0 when running tests.
> Thanks

i also meet the problem when training on cityscapes dataset,do you have sloved this problem,thanks
